Makefile

```diff
@@ -0,0 +1,17 @@
+all: install install-pipenv test
+
+install:
+	sudo apt-get install postgresql-13 \
+		postgresql-client-13 \
+		libhdf5-serial-dev \
+		libnetcdf-dev \
+		libspatialite-dev \
+		postgresql-12-postgis-3
```

Comment:
I still don't understand why a postgis for postgresql _12_ is being installed for a postgresql _13_ setup. Still less why it works, which it apparently does. 

This fails as an installation procedure on my workstation, BTW.

Reply:
Hmm I just grabbed the installation steps from the ci workflow, I didn't even notice the 12. I'll have to fix that.